app/assets/stylesheets/application.sass.scss

```diff
@@ -3,3 +3,4 @@ $govuk-new-link-styles: true;
 $govuk-assets-path: "/";
 
 @import "govuk-frontend/govuk/all";
+@import "panel-grey.scss";
```

Comment:
Ah, hang on. If we import `panel-grey` into the main stylesheet here, then we actually end up putting the new CSS rule onto the page _twice_ - once in the application.css which gets included in the `head` and once in the panel-grey.css that gets included in the body of `referrals/show.html.slim`.

So I think either you remove this import statement or you remove the `stylesheet_link_tag` in the view. As I understand it, `<link rel="stylesheet">` elements in the HTML body are valid, but not considered best practice, but equally, having this CSS rule in the main stylesheet means it's a tiny bit of extra redundant CSS getting loaded on every page, so I'm not sure which is the best option - I reckon either would be fine.

Reply:
I ended up leaving the import statement in and removing the stylesheet from the view.